include/staff/ticket-view.inc.php

```diff
@@ -434,6 +434,7 @@ class="icon-folder-close-alt icon-fixed-width"></i> %s</a></li>',
                              echo sprintf('<span><a class="manage-collaborators preview"
                                     href="#thread/%d/collaborators/1"><span id="t%d-recipients"><i class="icon-group"></i> (%s)</span></a></span>',
                                     $ticket->getThreadId(),
+                                    __('Manage Collaborators'),
```

Comment:
This will cause issues. There are only three replacement variables (`%d`, `%d`, `%s`) yet you are passing 4 values.

Reply:
Args, this line should not in the PR. 🙈 Looks like I don't disabled the change in the Github Desktop client. Will try to fix it. I am still a Git beginner.

Reply:
Create the new PR #5803 without the line.